src/assets/action.scss

```diff
@@ -75,6 +75,10 @@
 			.material-design-icon__svg {
 				vertical-align: middle;
 			}
+			body[dir=rtl] &.arrow-left-icon,
+			body[dir=rtl] &.chevron-right-icon {
+				transform: scaleX(-1);
+			}
```

Comment:
Not sure about this part. It mirrors all the arrow-like icons in NcAction button, link, text elements.

Reply:
These two arrow icons are used to indicate a direction within the interface.

One way to resolve this issue is to modify the .vue file that sets the icons’ direction based on the interface mode. Alternatively, we can flip these icons horizontally using CSS, which is the approach I took.

Reply:
I agree with @ShGKme 

Also I am not sure this should be handled by the library, because there might be use cases for left and right like the physical direction. It should probably be handled by the app instead.
Maybe we should either provide something like `NcIconArrowStart` and `NcIconArrowEnd` or a directive to flip if is RTL like `<IconArrayLeft v-rtl-icon />`.

Reply:
What if we use NcPopover and NcActionButton to create something like NcContextMenu which handles submenus and draws arrows if menu item has children.